.rubocop.yml

```
@@ -0,0 +1,30 @@
+
```

we should probably add a ruby target version like so:
```suggestion
AllCops:
  TargetRubyVersion: 2.3
```

I will get that into the `skel` to make it so new plugins get it. This helps us not accidentally introduce changes that do not work in say ruby `2.3` and were introduced in `2.4`.